be/CMakeLists.txt

```diff
@@ -241,6 +241,9 @@ set_target_properties(brotlienc PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/l
 add_library(zstd STATIC IMPORTED)
 set_target_properties(zstd PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libzstd.a)
 
+add_library(streamvbyte STATIC IMPORTED)
+set_target_properties(streamvbyte PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libstreamvbyte.a)
```

Comment:
remember to update `thirdparty/build-thirdparty.sh` and `thridparty/download-thirdparty.sh` if you import a new library

Reply:
done